leakcanary-analyzer/src/main/java/com/squareup/leakcanary/HeapAnalyzer.java

```diff
@@ -76,6 +79,8 @@ public AnalysisResult checkForLeak(File heapDumpFile, String referenceKey) {
       HprofBuffer buffer = new MemoryMappedFileBuffer(heapDumpFile);
       HprofParser parser = new HprofParser(buffer);
       Snapshot snapshot = parser.parse();
+      // reduces memory pressure from hprof bloat added in Marshmallow
```

Comment:
Moves this comment to javadoc of method.

Rename to `deduplicateGcRoots()` (or something else, but use full words.